features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/SyslogConnection.java

```diff
@@ -160,6 +162,12 @@ public SyslogProcessor call() {
         try {
             LOG.debug("Converting syslog message into event ({} bytes)", m_bytes.remaining());
 
+            try {
+                SyslogdIPMgrJDBCImpl.getInstance().dataSourceSync();
```

Comment:
This code has been updated and its also now handled such a way that this sync will be called only once during loading and wont be called again till opennms is stopped.